converter/src/main/java/gov/cms/qpp/conversion/segmentation/QrdaScope.java

```diff
@@ -57,11 +57,12 @@
 	}
 
 	public static QrdaScope getInstanceByName(String name) {
-		Optional<QrdaScope> found = Arrays.stream(QrdaScope.values())
-				.filter(inst -> inst.name().equals(name))
-				.findFirst();
-
-		return found.orElse(null);
+		Objects.requireNonNull(name, "name");
+		String match = name.trim().replace(' ', '_').toLowerCase();
+		return Arrays.stream(QrdaScope.values())
+				.filter(value -> value.name().toLowerCase().equals(match))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1449)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)